package-lock.json

```diff
@@ -20581,7 +20581,7 @@
     },
     "plugins/pa11y": {
       "name": "@dotcom-tool-kit/pa11y",
-      "version": "0.3.2",
```

Comment:
**question:** Is this change required in this commit?

Reply:
this change happens automatically after running `npm install`. release-please isn't able to update the lockfiles when it merges releases so it will have to get updated manually instead. I'd consider it a minor change that we can smuggle into other PRs/commits to avoid us constantly manually updating it in dedicated PRs (and remembering to exclude it from staging in other unrelated commits.)